pokemongo_bot/inventory.py

```diff
@@ -100,7 +100,27 @@ class Items(_BaseInventoryComponent):
     STATIC_DATA_FILE = os.path.join(_base_dir, 'data', 'items.json')
 
     def count_for(self, item_id):
-        return self._data[item_id]['count']
+        return self._data[item_id].get('count', False)
+
+    def get_space_used(self):
+        """
+        Counts the space used in item inventory.
+        :return: The space used in item inventory.
+        :rtype: int
+        """
+        itemcount = 1
+        for item in self._data:
+                itemcount += self.count_for(item)
```

Comment:
if `self._data[item_id].get('count', False)` fails and return `False`, then this line fails. It should only add if `self.count_for(item)` is a truthy value.